x509/src/certificate/document.rs

```diff
@@ -1,97 +0,0 @@
-//! CertificateDocument implementation
```

Comment:
@carl-wallace FYI, ended up removing this along with all of the format-specific `*Document` types. Hope the `Document` type works for your use cases